main.css

```diff
@@ -0,0 +1,465 @@
+@import url("https://fonts.cdnfonts.com/css/cocogoose");
+@import url("https://fonts.cdnfonts.com/css/Lato");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.top-links-container {
+  flex-direction: row;
+  width: 100%;
+  height: 40px;
+  background-color: #272a31;
+}
+
+.top-links {
+  align-items: center;
+  justify-content: end;
+  font-family: 'Lato', sans-serif;
+  padding: 10px 55px 0 0;
+}
+
+li a {
+  text-decoration: none;
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-size: 1rem;
+}
+
+.top-links > a {
+  float: right;
+  color: #d3d3d3;
+  text-decoration: none;
+  margin: auto 5px;
+}
+
+header {
+  background-color: #fff;
+  font-family: 'Lato', sans-serif;
+}
+
+nav {
+  padding: 20px 0;
+}
+
+.nav-toggle {
+  position: absolute;
+  top: 0.5rem;
+  left: 1rem;
+  display: none;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 20px;
+  padding: 0 1rem;
+}
```

Comment:
[Optional] I suggest that you make the mobile menu to be fixed and not scroll. This will help the user know that they have a pending action to take in order to close it.

![image](https://user-images.githubusercontent.com/20554583/197339624-6d0f363d-4f7a-46df-a326-17e6e947efb9.png)